@xen-orchestra/fs/src/abstract.spec.js

```diff
@@ -52,6 +52,18 @@ describe('createReadStream()', () => {
   })
 })
 
+describe('getInfo()', () => {
+  it(`throws in case of timeout`, async () => {
```

Comment:
Use straight quotes when you don't need interpolation.

Reply:
All methods are with string interpolation, do I change everywhere?

Reply:
Nope just here for the moment.